pkg/filesystem/BUILD.bazel

```diff
@@ -13,6 +13,7 @@ go_library(
         "local_directory_linux.go",
         "local_directory_unix.go",
         "local_directory_windows.go",
+        "mount_unix.go",
```

Comment:
Wanted to move the syscall cruff out of the way. We should not need so much code.